app/assets/javascripts/components/import_ratings/import_ratings_controller.js

```diff
@@ -23,23 +24,32 @@ angular.module('QuepidApp')
           size: 'lg',
           resolve:      {
             theCase: function() {
-              return ctrl.acase;
+              return caseSvc.getSelectedCase(); //TODO not sure why this is needed for import snapshot to work?
+            },
+            querySnapshotSvc: function() {
```

Comment:
Not positive, but because of dependency injection, I don't think you need htise...  they are injected into the modal controller directly...